content/v3/pulls/comments.md

```diff
@@ -62,7 +62,7 @@ Name | Type | Description
 Name | Type | Description
 -----|------|--------------
 `body`|`string` | **Required**. The text of the comment
-`commit_id`|`string` | **Required**. The SHA of the commit to comment on.
+`commit_id`|`string` | **Required**. The SHA of the latest commit of the pull request to comment on.
```

Comment:
:wave: Hi @tomasbjerre, I agree that this is a little confusing, but in some cases users may want to comment on a previous version of the PR via the API. In order to do so, they will need to pass a `commit_id` that's not equal to the "latest commit of the pull request". What do you think about adding a note at the bottom of the input table? Something to the tune of:

> **Note**: When passing the `commit_id`, use the SHA of the latest commit in the pull request or your comment may appear as "outdated" if the specified position has been modified in a subsequent commit.